### ISSUE TITLE
Route for particular post

### ISSUE THREAD

**Issue by alexcwatt** [rubysherpas/forem#557]
I am working on a custom email notifier and tried using the following to get a URL for a particular post that needed to be approved:

```
<%= link_to "Post on #{post.topic}",
forem.forum_topic_post_url(post.forum, post.topic, post) %>
```

It returns a link, but the link does not work... If `/edit` is appended to the URL, it does work.

Making this work will require defining the `show` action for posts. I propose redirecting to the topic path with the appropriate page parameter.

Does this sound like a good approach? Or maybe I'm missing a feature that forem already has?

Thanks.

**Comment by radar** [comment]
Hey @alexcwatt,

I think having either a show action that shows the post (maybe even within the context of the topic) and provides the moderator the ability to approve the post or mark it as spam is a good idea. 

Could you please submit a PR which implements this feature?

**Comment by alexcwatt** [comment]
Sure @radar. It may be a while, but it's on the list.

**Comment by radar** [comment]
Looks like this was solved with #576.
